Tests/PcppTestFramework/PcppTestFramework.h

```diff
@@ -6,20 +6,44 @@
 #include "memplumber.h"
 #include "PcppTestFrameworkCommon.h"
 
-#define _PTF_PRINT_TYPE(val) "[" << val << "]"
-#define object_no_str_PTF_PRINT_TYPE(val) "'" << #val << "'"
-#define hex_PTF_PRINT_TYPE(val) "0x" << std::hex << val
-#define enum_PTF_PRINT_TYPE(val) #val << "[" << val << "]"
-#define ptr_PTF_PRINT_TYPE(val) #val << "[" << val << "]"
+#define _PTF_PRINT_TYPE_ACTUAL(exp, val) val
+#define _PTF_PRINT_TYPE_EXPECTED(exp, val) val
+#define hex_PTF_PRINT_TYPE_ACTUAL(exp, val) "0x" << std::hex << +val << std::dec
+#define hex_PTF_PRINT_TYPE_EXPECTED(exp, val) "0x" << std::hex << +val << std::dec
+#define enum_PTF_PRINT_TYPE_ACTUAL(exp, val) "enum[" << val << "]"
+#define enum_PTF_PRINT_TYPE_EXPECTED(exp, val) exp << "[" << val << "]"
+#define ptr_PTF_PRINT_TYPE_ACTUAL(exp, val) exp << "[ptr: " << val << "]"
+#define ptr_PTF_PRINT_TYPE_EXPECTED(exp, val) exp << "[ptr: " << val << "]"
+#define enumclass_PTF_PRINT_TYPE_ACTUAL(exp, val) "enum[" << +static_cast<std::underlying_type<decltype(val)>::type>(val) << "]"
+#define enumclass_PTF_PRINT_TYPE_EXPECTED(exp, val) exp << "[" << +static_cast<std::underlying_type<decltype(val)>::type>(val) << "]"
+
+#define PTF_PRINT_ASSERTION(severity, op) \
+	std::cout << std::left << std::setw(35) << __FUNCTION__ << ": " \
+    << severity \

```

Comment:
nit: alignment (not checked probably space to tab)

Reply:
Good catch! Fixed in: https://github.com/seladb/PcapPlusPlus/pull/1019/commits/558a259db9679488400b52bee0e6322da55256c2